File: sample/src/main/java/com/lynx/wind/recyclersample/holder/SimpleHolder.java
package com.lynx.wind.recyclersample.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.lynx.wind.recyclersample.R;
import com.lynx.wind.recyclersample.model.User;

public class SimpleHolder extends RecyclerView.ViewHolder {
    public SimpleHolder(View itemView) {
        super(itemView);
    }

    public void bind(User data){
        ((TextView)itemView.findViewById(R.id.txt)).setText(data.getName());
    }
}

File: sample/src/main/java/com/lynx/wind/recyclersample/holder/SimpleHolder.kt
package com.lynx.wind.recyclersample.holder

import android.support.v7.widget.RecyclerView
import android.view.View
import com.lynx.wind.recyclersample.model.User
import kotlinx.android.synthetic.main.item_user.view.*

class SimpleHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
    fun bind(data: User){
        itemView.txt.text = data.name
    }
}